helmfile.d/stacks/tekton.yaml

```
@@ -0,0 +1,15 @@
+---
+templates:
+  tekton-pipelines:
+    condition: ck8sManagementCluster.enabled
+    name: tekton-pipelines
+    chart: charts/tekton-pipelines
+    installed: {{ .Values | get "tektonPipelines.enabled" false }}
+    version: 0.1.0
+    namespace: tekton-pipelines
+    labels:
+      app: tekton
+    needs:
+      - kube-system/admin-namespaces
+    values:
+      - values/tekton.gotmpl
```

```suggestion
  tekton-pipelines:
    disableValidationOnInstall: true
    condition: ck8sManagementCluster.enabled
    installed: {{ .Values | get "tektonPipelines.enabled" false }}
    namespace: tekton-pipelines
    name: tekton-pipelines
    labels:
      app: tekton
    chart: charts/tekton-pipelines
    version: 0.1.0
    needs:
      - kube-system/admin-namespaces
      - monitoring/kube-prometheus-stack # creates servicemonitors and prometheusrules
    values:
      - values/tekton.gotmpl
```
I have a convention on the order here to make it more harmonious :smile: 

Though the `disableValidationOnInstall: true` is required whenever a release installs CRs of CRDs installed in Apps, and the added need to ensure those are installed before tekton.